packages/webamp-modern-2/src/maki/interpreter.ts

```diff
@@ -294,25 +294,62 @@ class Interpreter {
           let argCount: number = klass.prototype[methodName].length;
 
           const methodDefinition = getMethod(guid, methodName);
-          assert(
-            argCount === (methodDefinition.parameters.length ?? 0),
-            `Arg count mismatch. Expected ${
-              methodDefinition.parameters.length ?? 0
-            } arguments, but found ${argCount} for ${klass.name}.${methodName}`
-          );
+          if (methodName.toLowerCase() != "init") {
+            assert(
+              argCount === (methodDefinition.parameters.length ?? 0),
+              `Arg count mismatch. Expected ${
+                methodDefinition.parameters.length ?? 0
+              } arguments, but found ${argCount} for ${
+                klass.name
+              }.${methodName}`
+            );
+          }
 
           const methodArgs = [];
           while (argCount--) {
             const a = this.stack.pop();
             methodArgs.push(a.value);
           }
           const obj = this.stack.pop();
+          if (
+            !obj.value &&
+            ((klass.name || "").toLowerCase() == "winampconfig" ||
+              (klass.name || "").toLowerCase() == "winampconfiggroup" ||
+              (klass.name || "").toLowerCase() == "configclass")
+          ) {
+            obj.value = new klass();
+          }
           assert(
             (obj.type === "OBJECT" && typeof obj.value) === "object" &&
               obj.value != null,
             `Guru Meditation: Tried to call method ${klass.name}.${methodName} on null object`
           );
-          let value = obj.value[methodName](...methodArgs);
+          // let value = obj.value[methodName](...methodArgs);
+          let value = null;
+          try {
+            if (obj.value[methodName]) {
+              value = obj.value[methodName](...methodArgs);
+            } else {
+              value = obj.value.constructor[methodName](...methodArgs);
+            }
+          } catch (err) {
+            value = null;
+
+            try {
+              const fun = (
+                obj.value[methodName]
+                  ? obj.value[methodName]
+                  : obj.value.constructor[methodName]
+              ).bind(obj.value);
+              value = fun(...methodArgs);
+            } catch (err) {
+              console.warn(
+                `error call: ${klass.name}.${methodName} args:${methodArgs}`,
+                `err: ${err.message} obj: ${JSON.stringify(obj)}`
+              );
+              value = null;
+            }
+          }
```

Comment:
I think you replied to a question I had here (I think I saw a notification) but I don't see it here.

Reply:
yes, it was about if we could show a warn (on catch error) instead of silent.
But I said that I catch error in a nested try/catch, so I do not want to give a false-positive warning,
I mean the warning is only shown when there is finally a real problem.